tests/integration/test_notes.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from conftest import is_valid
+from models.notes import NotesModel
+
+
+@pytest.mark.usefixtures("client_class", "empty_test_db")
+class TestCreate:
+    def setup(self):
+        self.endpoint = "/api/tickets"
+        self.newNote = {"text": "We don't need no water"}
+
+    def test_creates_note_through_endpoint(
+        self, valid_header, create_admin_user, create_tenant, create_ticket
+    ):
+        create_admin_user()
+        create_tenant()
+        validTicketID = create_ticket().json()["id"]
+
+        response = self.client.post(
+            f"{self.endpoint}/{validTicketID}/notes",
+            json=self.newNote,
+            headers=valid_header,
+        )
+
+        assert response.json == NotesModel.find(response.json["id"]).json()
```

Comment:
This assertion seems a bit odd. I don't like using the response on both sides of the equality check. Is there another way we can assert our note was created?

Reply:
We could hardcode like this but that seems like it will make the test more fragile.  
```python
        assert response.json == NotesModel.find(1).json()
```


I could pull the number from last note added to the ticket.
```python
        note_id = TicketModel.find(validTicketID).json()["notes"][-1]["id"]
        assert response.json == NotesModel.find(note_id).json()
```

Does one of these solutions work better?  I'll keep thinking about this, see if I can come up with a different solution.

Reply:
I'd go with your second approach. Although, we already have the ticket object so there is not need to ask the database for it. 

Your test could be a bit more concise and look something like this:
```
        ticket = create_ticket()

        response = self.client.post(
            f"{self.endpoint}/{ticket.id}/notes",
            json=self.payload,
            headers=valid_header,
        )

        assert response.json == ticket.notes[-1].json()
```

Reply:
That makes sense.  I made those changes and pushed a new commit.

Reply:
Technically we're still asking the db for the notes, but doing so through the association. Just a bit more cleaner and easier to read.